client/src/app/shared/shared-main/video/video.service.ts

```diff
@@ -398,9 +398,14 @@ export class VideoService implements VideosProvider {
       }
     ]
 
-    return base
+    const videoPrivacies = base
       .filter(o => !!privacies.find(p => p.id === o.id)) // filter down to privacies that where in the input
       .map(o => ({ ...privacies[o.id - 1], ...o })) // merge the input privacies that contain a label, and extend them with a description
+
+    return {
+      firstStepPrivacyId: videoPrivacies.find(p => p.id === VideoPrivacy.PUBLIC).id || videoPrivacies[0].id,
```

Comment:
Put `VideoPrivacy.PUBLIC` as a default arg so we explicitly choose the default privacy in components

Reply:
`firstStepPrivacyId` -> `defaultPrivacyId`